app/Http/Controllers/API/ServerController.php

```diff
@@ -122,7 +122,7 @@ public function view(Request $request, $id)
                 'pack'
             )->first();
             if ($request->input('daemon') === 'true') {
-                $node = Models\Node::findOrFail($server->node);
+                $node = Models\Node::findOrFail($server->node_id);
```

Comment:
Just do it! 😂 
Before merging, this PR should be fully done with 0.6.0 (user AND admin side)